src/app/utils/dateFormatter.ts

```diff
@@ -1,36 +1,67 @@
-import formatDistanceStrictDateFns from 'date-fns/formatDistanceStrict'
+import { formatDistanceToNow as dateFnsFormatDistanceToNow } from 'date-fns'
 import locale from 'date-fns/locale/en-US'
 
 const dateFormat = new Intl.DateTimeFormat()
 
 export const intlDateFormat = (date: Date | number) => dateFormat.format(date)
 
 const formatDistanceLocale = {
-  lessThanXSeconds: '{{count}}s',
-  xSeconds: '{{count}}s',
-  halfAMinute: 's',
-  lessThanXMinutes: '{{count}}m',
-  xMinutes: '{{count}}m',
-  aboutXHours: '{{count}}h',
-  xHours: '{{count}}h',
-  xDays: '{{count}}d',
-  aboutXWeeks: '{{count}}w',
-  xWeeks: '{{count}}w',
-  aboutXMonths: '{{count}}m',
-  xMonths: '{{count}}m',
-  aboutXYears: '{{count}}y',
-  xYears: '{{count}}y',
-  overXYears: '{{count}}y',
-  almostXYears: '{{count}}y',
+  lessThanXSeconds: '{{count}}sec',
```

Comment:
not related to PR. Should we keep all suffixes and ` ago` etc in 18next ? if yes, I will create ClickUp task.

Reply:
Probably, abbreviations seems fine. But `ago` and `in` would make sense to put them into translations.

Reply:
can we replace all of this with https://date-fns.org/v2.30.0/docs/intlFormatDistance style: narrow

Reply:
Good catch, will try it out.